README.md

```diff
@@ -1,46 +1,90 @@
-# OpenGloves Driver [![Steam Badge]][Steam] [![Discord Badge]][Discord]
 
-A **SteamVR (OpenVR)** driver for VR Gloves and DIY Hardware.<br>
+<br>
 
-## Download
-[![Steam Badge]][Steam]
+<div align = center>
 
-OpenGloves is available on **[Steam]**,<br>
-where it comes with additional tools for configuring driver related features.
+[![Badge Steam]][Steam]   
+[![Badge Discord]][Discord]
 
-## Features
+<br>
 
-### Driver Features
+# OpenGloves Driver
 
-- **Force Feedback**
+***SteamVR*** */ **OpenVR** driver for **VR Gloves** and **DIY Hardware**.*
 
-- **Full Finger Tracking** <br>
-    *Splay Support* <br>
-    *Individual Joint Support*
+<br>
+<br>
+
+[![Button Install]][Install]   
+[![Button Building]][Building]   
+[![Button Limits]][Limits]   
+[![Button Compatibility]][Compatibility]
+
+</div>
+
+<br>
+<br>
+
+## Driver Features
+
+<br>
 
 - **Tracker / Controller Positioning & Offsetting** <br>
 
-- **Button / Joystick Inputs** <br>
-    *➜ Trigger | A | B* <br>
-    *➜ Joystick | X | Y | Click*
+    <br>
+    
+-   **Multiple Communication Methods** <br>
+    
+    - *Bluetooth Serial*
+    - *Named Pipes*
+    - *Serial USB*
+    
+    <br>
+
+- **Full Finger Tracking** <br>
+    
+    - *Individual Joint Support*
+    - *Splay Support*
+
+    <br>
 
-- **Multiple Communication Methods** <br>
-    *- Bluetooth Serial* <br>
-    *- Named Pipes* <br>
-    *- Serial USB*
+-   **Button / Joystick Inputs**
 
-### Additional Tools
+    - <kbd>      Trigger      </kbd>  <kbd>  A  </kbd>  <kbd>  B  </kbd> <br>
+    - <kbd>  Joystick  </kbd>  <kbd>  X  </kbd>  <kbd>  Y  </kbd>  <kbd>  Click  </kbd>
```

Comment:
Not a big fan of what these look like... I think how they were previously was fine.

Reply:
That's unfortunate, kbd is even made for that~